src/EventListener/GeneratePageListener.php

```diff
@@ -60,7 +60,9 @@ public function __invoke(PageModel $pageModel, LayoutModel $layout, PageRegular
         if (!$layout->addEncore) {
             return;
         }
-        $this->addEncore($pageModel, $layout, $pageRegular);
+        if (!isset($this->bundleConfig['use_contao_template_variables']) || true !== $this->bundleConfig['use_contao_template_variables']) {
```

Comment:
why you ask 

true !== $this->bundleConfig['use_contao_template_variables']

and not just !$this->bundleConfig['use_contao_template_variables']  or false ===

Reply:
Cause it should only be applied if it's an boolean true ;)